src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerContext.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Diagnostics;
+
+namespace System.Text.Json.Serialization.Metadata
+{
+    /// <summary>
+    /// Provides metadata about a set of types that is relevant to JSON serialization.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract partial class JsonSerializerContext
+    {
+        internal JsonSerializerOptions? _options;
```

Comment:
Why does this need to be nullable?

Reply:
It shouldn't be. Thanks.

Reply:
There's an intermediate state where the context may be initialized without options, and no options are set either until `JsonSerializerOptions.AddContext` is called, or `JsonSerializerContext.Options` is called. The nullability reflects this possibility.